### PR TITLE
[IMP] Inventory: update for 16

### PR DESCRIPTION
**Pull request by larm-odoo**
Updated 3 sentences- no need to click 'Edit' in v 16, no need to click 'Save', and the create button is now a 'New' button.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @odoo/inventory-doc-review - this is ready for a look whenever you get a chance. Thank you!

**Review by hojo-odoo — COMMENTED** [review]
Hey @larm-odoo,

These changes look good. You can tag @samueljlieber for technical review.

Thanks,
hojo

File: content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_one_step.rst [hunk 1]
```diff
@@ -90,7 +89,7 @@ Create a sales order
 --------------------
 
 Start my navigating to the main :menuselection:`Sales` app dashboard, and make a new quote by
-clicking :guilabel:`Create`. Then select (or create) a :guilabel:`Customer` from the drop-down
+clicking :guilabel:`New`. Then select (or create) a :guilabel:`Customer` from the drop-down
 field, add a storable :guilabel:`Product` that is in stock to the order lines, and click
 :guilabel:`Confirm` to finalize the quotation as a sales order.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_one_step.rst:94]: Early line break
```suggestion
clicking :guilabel:`New`. Then select (or create) a :guilabel:`Customer` from the drop-down field,
add a storable :guilabel:`Product` that is in stock to the order lines, and click
:guilabel:`Confirm` to finalize the quotation as a sales order.
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo ✌️nice job! I have just one correction, please implement this before tagging for final review, approving now!

**Comment by larm-odoo** [comment]
Thanks @samueljlieber - sending this quick doc update to @StraubCreative !

**Pull request closed without merging**